src/modules/airspeed_selector/AirspeedValidator.hpp

```diff
@@ -181,8 +181,10 @@ class AirspeedValidator
 	bool _first_principle_check_failed{false}; ///< first principle check has detected failure
 	float _aspd_fp_t_window{0.f}; ///< time window for first principle check
 	FilteredDerivative<float> _IAS_derivative; ///< indicated airspeed derivative for first principle check
-	AlphaFilter<float> _throttle_filtered; ///< filtered throttle for first principle check
+	static constexpr float throttle_filter_time_constant{0.5f};
```

Comment:
Please read again the link I shared in https://github.com/PX4/PX4-Autopilot/pull/24522#discussion_r2001491165